src/modules/search/components/SearchBar/SearchBar.css

```diff
@@ -1,21 +1,34 @@
 @import '../../../../../setup/react/app/mixins.css';
 
 .wrapper {
-  margin-top: 5px;
   min-width: 456px;
   box-sizing: border-box;
   display: flex;
   align-items: center;
+  height: 100%;
 
   & .icon {
+    display: flex;
     left: 10px;
   }
 
   & input {
     border-width: 0 !important;
-    border-radius: var(--border-radius-box) !important;
-    background: rgba(255, 255, 255, 0.12);
-    color: var(--color-white);
+    border-radius: 40px;
+    background: #2e3855;
```

Comment:
@oskarleonard Since this colour is used both in light and dark modes, then it should be created as a variable and that way, the dark mode variant won't be exclusively handled here

Reply:
You mean like this? https://github.com/LiskHQ/lisk-desktop/pull/4647/commits/e47f9c6de125684e09b2060960c3c164455562f9